src/scripts/utils/swRegister.es6

```diff
@@ -0,0 +1,5 @@
+if ('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('serviceWorker.js').then((registration) => {
+		console.log(registration)
+	})
```

Comment:
Když se to nezaloguje tak to nikdy nezjistíš. (I když sentry je mnohonásobně lepší...)